flipperdevices/Flipper-Android-App
# move remote controls to infrared

**Background**

Right now, `remote-controls` add menu locate under experimental section. Currently its debug-only feature, so it should be enabled by flag inside debug section

**Changes**

- Remote `remote-controls` from experimental section
- Add flag for show remote controls
- Move add-remote inside infrared archive category

**Test plan**

- Open dev settings
- Set show remote controls
- Open infrared archive and see remote controls
- Do it again with disabled remote controls
- See no add remote is shown on infrared archive
